Ros1 manager bt create basic trees (#104)
husarion/panther_ros: panther_manager/README.md

## ROS Nodes

> ### manager_bt_node.cpp

Reviewer: I would name this after executable, not file. `.py` is an executable, but in this case I would leave it without cpp

- `~shutdown_timeout` [*float*, default: **15.0**]: time in seconds to wait for graceful shutdown. After timeout power will be cut from all devices.
- `~update_charging_anim_step` [*float*, default: **0.1**]: percentage value representing a step for updating the charging battery animation.

### system_status_node.py

Publishes stats status of the built-in computer. Stats include CPU utilization and temperature, as well as disc and RAM usage.

#### Publishes

- `/panther/system_status` [*panther_msgs/SystemStatus*]: information about internal computer CPU temperature, utilization, disc and RAM usage.

## BehaviorTree 

### Nodes

#### Actions

- `CallSetBoolService` - allows calling standard **std_srvs/SetBool** ROS service.
- `CallSetLedAnimationService` - allows calling custom type **panther_msgs/SetLEDAnimation** ROS service.
- `CallTriggerService` - allows calling standard **std_srvs/Trigger** ROS service.
- `ShutdownHostsFromFile` - allows to shutdown devices based on YAML file. Returns FAILURE only when YAML file is incorrect or a path to the file does not exists. If it fails to execute command in a remote host node will proceed tothe next device from the list.

> - `ShutdownSingleHost` - allows to shutdown single device. Will return SUCCESS only when device can be reached and the command was executed.

Reviewer: ```suggestion
- `CallSetBoolService` - allows calling standard **std_srvs/SetBool** ROS service.
- `CallSetLedAnimationService` - allows calling custom type **panther_msgs/SetLEDAnimation** ROS service.
- `CallTriggerService` - allows calling standard **std_srvs/Trigger** ROS service.
- `ShutdownHostsFromFile` - allows to shutdown devices based on YAML file. Returns `FAILURE` only when YAML file is incorrect or a path to the file does not exists. If it fails to execute command in a remote host node will proceed tothe next device from the list.
- `ShutdownSingleHost` - allows to shutdown single device. Will return `SUCCESS` only when device can be reached and the command was executed.
```
Nodes also descriptions miss available ports with their types

Reviewer: There is still description of ports of nodes missing

#### Decorators

> - `TickAfterTimeout` - will skip child before specified time has passed. Can be used to control frequency at witch given behavior in the tree is triggered.

Reviewer: ```suggestion
- `TickAfterTimeout` - will skip child before specified time has passed. Can be used to specify frequency at witch node or subrtee is triggered.
```

### Trees

- `Lights` - tree responsible for scheduling animations displayed on LED panels, based on the Husarion Panther robot's system state.
- `Safety` - tree responsible for monitoring the Panther robot's state and handling safety measures, such as  cooling the robot in case of high CPU or battery temperature.

> - `Shutdown` - tree responsible for gracefull shutdown of robot components and the built-in computer.

Reviewer: ```suggestion
- `Shutdown` - tree responsible for gracefull shutdown of robot components, user computers and the and the built-in computer.
```

> - `Shutdown` - this tree is responsible for gracefull shutdown of robot components and the built-in computer.

Reviewer: ```suggestion
- `Lights` - tree responsible for scheduling animations displayed on LED panels, based on the Husarion Panther robot's system state.
- `Safety` - tree responsible for monitoring the Panther robot's state and handling safety measures, such as  cooling the robot in case of high CPU or battery temperature.
- `Shutdown` - tree responsible for gracefull shutdown of robot components and the built-in computers.
```

> - `Shutdown` - tree responsible for gracefull shutdown of robot components, user computers and the built-in computer.

Reviewer: There is still no information about black board input and output ports and no information about constants like animation IDs and so on. Can you at least make draft to be finished in a separate task?
